File: Bank.Frontend/src/app/Components/accounts/accountDto.ts
export interface  AccountDto {

  id: number,
  customerId: number,
  accountNo: string,
  accountType: number,
  status: number,
  balance: number,
  currency: string,
  opendOnUtc: Date,
  closedOnUtc: Date,
  rowVersion: string
}
export interface withCustomerDto {

  id: number;
  accountNo: string,
  balance:number,
  customerId: number,
  customerName: string,
  customerEmail: string

}
export interface CreateAccountDto {
  id:number;
  accountNo: string;
  accountType: number;
  status: number;
  balance: number;
  currency: string;
  opendOnUtc: Date;
  closedOnUtc: Date;
  rowVersion: string;
  customerName: string;   
  customerEmail: string;  
  customerId:number;
}

export interface updateAccountDto{
id: number,
  customerId: number,
  accountNo: string,
  accountType: number,
  status: number,
  balance: number,
  currency: string, 
   opendOnUtc: Date,
  closedOnUtc: Date,
  rowVersion: string,
  pincode:string

}


export interface CheckBalanceDto {
  customerId: number;
  pinCode: string;
}

export interface AccountBalanceResponse {
  accountId: number;
  accountNo: string;
  customerName: string;
  accountType: number;
  balance: number;
}

File: SRC/Core/Bank.Application/Features/Account/Dtos/AccountDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bank.Application.Features.Account.Dtos
{
    public class AccountDto
    {
        public int Id { get; set; }
        public int CustomerId { get; set; }
        public string Account { get; set; }
        public int AccountType { get; set; }
        public int Status { get; set; }
        public decimal Balance { get; set; }
        public string Currency { get; set; }
        public DateTime OpendOnUtc { get; set; }
        public DateTime? ClosedOnUtc { get; set; }
        public string RowVersion { get; set; }
    }
}
